ext/pdo_firebird/tests/bug_77863.phpt

```diff
@@ -0,0 +1,139 @@
+--TEST--
+PDO_Firebird: Bug #76488 PDO Firebird does not support boolean datatype in input parameters
+--SKIPIF--
+<?php if (!extension_loaded('interbase') || !extension_loaded('pdo_firebird')) die('skip');
```

Comment:
Is it really necessary to have ext/interbase loaded? If so that would be a problem with https://wiki.php.net/rfc/deprecate-and-remove-ext-interbase.

Reply:
Most likely no. I just copied this piece from other pdo_firebird tests.

Reply:
Thanks! That is something @KalleZ should be aware of. :)

Reply:
Unfortunately, all the firebird_pdo tests use the testdb.inc file in which the test database is created. For this, the interbase extension is used. Due to the fact that this extension was decided to be removed, it is necessary to rewrite the test database initialization procedure using only firebird_pdo, but this is a separate task. I'll see if you can do it.

Reply:
I fixed this in both `PHP-7.4` and `master`, so the `--SKIPIF--` section should now just look like:

```php
<?php require('skipif.inc'); ?>
```

Please check the `testdb.inc` file for testing as it now operates on environment variables. Relevant commits:
https://github.com/php/php-src/commit/c9599c1c726e3bce8ce5a80c1ac8ee2c81bffb61
https://github.com/php/php-src/commit/1c893b89bdb2b8bd5608c14cd0930727db6ba409

The first one does the ground work and the second one an enhancement on top of the first one.

I was not able to test on Unix, so this is only tested on Windows but it works. If there is any issues then I'll gladly accept a PR as I'm personally not familiar with proper configuration of FireBird/InterBase.